vars/withCloudEnv.groovy

```diff
@@ -26,7 +26,7 @@
 def call(Map args = [:], Closure body) {
   log(level: 'INFO', text: 'withCloudEnv')
   def cluster = args.containsKey('cluster') ? args.cluster : error('withCloudEnv: cluster parameter is required.')
-  def secret = "secret/observability-team/ci/test-clusters/${cluster}/ec-deployment"
+  def secret = "${getTestClusterSecret()}/${cluster}/ec-deployment"
```

Comment:
ec-deployment is no longer created

Reply:
I'll raise an issue to change the behaviour in a follow up. https://github.com/elastic/apm-pipeline-library/pull/1728 is required for this particular PR, since I made changes on the top to be able to move forward